package.json

```diff
@@ -1,11 +1,20 @@
 {
-  "name": "express-hw",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "volfram@ukr.net",
-  "license": "ISC"
+	"name": "express-hw",
+	"version": "1.0.0",
+	"description": "ExpressJS EPAM Learning",
+	"main": "index.js",
+	"type": "module",
+	"scripts": {
+		"start": "node index.js",
+		"dev": "nodemon --watch"
+	},
+	"author": "volfram@ukr.net",
+	"license": "ISC",
+	"dependencies": {
+		"body-parser": "^1.20.2",
+		"express": "^4.18.2",
+		"nodemon": "^3.0.1",
```

Comment:
The `nodemon` package should be in the devDependencies section because it is only used for development and there is no need to install this package in the production build.